src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -140,7 +154,15 @@ static bool templateNotify
     std::string key   = it->first;
     std::string value = it->second;
 
+    macroSubstitute(&key,   it->first, ce);
     macroSubstitute(&value, it->second, ce);
+
+    if (key == "")
```

Comment:
As far as I understand (I think we were talking about that at skype), blank value makes sense for underlying libcurl as a mean of removing a default header (e.g. User-Agent). Thus, in the case of headers, it makes sense a empty `value`.